test/server/cards/03_TWI/units/IndependentSenator.spec.ts

```diff
@@ -0,0 +1,39 @@
+describe('Independent Senator', function () {
+    integration(function (contextRef) {
+        describe('Independent Senator\'s ability', function () {
+            it('should heal 2 damage from a unit or a base', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['independent-senator']
+                    },
+                    player2: {
+                        groundArena: ['wampa', 'atst'],
+                        spaceArena: ['gladiator-star-destroyer', 'green-squadron-awing']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                function reset() {
+                    context.player1.moveCard(context.echo, 'hand');
+                    context.player2.passAction();
+                }
+
+                // Case 1: A groud areana with less than 4 power can be exhausted
```

Comment:
nit: we generally only use comments like this if we're doing multiple different scenarios in one case, in this one it's fine since the test case name already describes the one thing we're doing